ansible_collections/arista/avd/molecule/eos_cli_config_gen/intended/configs/host2.cfg

```diff
@@ -109,6 +109,75 @@ policy-map type pbr POLICY_DROP_THEN_NEXTHOP
 router bfd
    session stats snapshot interval dangerous 8
 !
+router bgp 65101
+   no bgp default ipv4-unicast
+   no bgp default ipv4-unicast transport ipv6
+   graceful-restart
+   no graceful-restart-helper
+   bgp route-reflector preserve-attributes
+   no bgp additional-paths receive
+   no bgp additional-paths send
+   bgp redistribute-internal
+   redistribute connected include leaked route-map RM-CONN-2-BGP
+   redistribute isis level-2 include leaked rcf RCF-CONN-2-BGP()
+   redistribute ospf match internal include leaked route-map RM_BGP_EVPN
+   redistribute ospf match external include leaked route-map RM_BGP_EVPN
+   redistribute ospfv3 match internal include leaked route-map RM-CONN-2-BGP
+   redistribute static route-map RM-STATIC-2-BGP
+   redistribute dynamic rcf RCF-CONN-2-BGP()
+   !
+   address-family evpn
+      no bgp additional-paths send
+      neighbor default encapsulation path-selection
+      neighbor EVPN-OVERLAY-PEERS activate
+      no neighbor MLAG-IPv4-UNDERLAY-PEER activate
+      neighbor default next-hop-self received-evpn-routes route-type ip-prefix inter-domain
+      host-flap detection expiry timeout 20 seconds
```

Comment:
This is not already covered by host1's `host-flap detection window 10 threshold 1 expiry timeout 3 seconds` under the same "address-family evpn"?

Reply:
Using fewer or different options may exercise different paths of the code.

Reply:
yes, so this will help to improve the coverage

Reply:
Rendering `host-flap detection window 10 threshold 1 expiry timeout 3 seconds` would already use all related parts of j2 template, so having another `host-flap detection expiry timeout 20 seconds` seems excessive

```jinj2
{%         elif router_bgp.address_family_evpn.evpn_hostflap_detection.enabled is arista.avd.defined(true) %}
{%             set hostflap_detection_cli = '' %}
{%             if router_bgp.address_family_evpn.evpn_hostflap_detection.window is arista.avd.defined %}
{%                 set hostflap_detection_cli = hostflap_detection_cli ~ ' window ' ~ router_bgp.address_family_evpn.evpn_hostflap_detection.window %}
{%             endif %}
{%             if router_bgp.address_family_evpn.evpn_hostflap_detection.threshold is arista.avd.defined %}
{%                 set hostflap_detection_cli = hostflap_detection_cli ~ ' threshold ' ~ router_bgp.address_family_evpn.evpn_hostflap_detection.threshold %}
{%             endif %}
{%             if router_bgp.address_family_evpn.evpn_hostflap_detection.expiry_timeout is arista.avd.defined %}
{%                 set hostflap_detection_cli = hostflap_detection_cli ~ ' expiry timeout ' ~ router_bgp.address_family_evpn.evpn_hostflap_detection.expiry_timeout ~ ' seconds' %}
{%             endif %}
{%             if hostflap_detection_cli != '' %}
      host-flap detection{{ hostflap_detection_cli }}
{%             endif %}
```